docs/ops.html

```diff
@@ -62,8 +62,8 @@
   <pre class="brush: bash;">
   &gt; bin/kafka-topics.sh --zookeeper zk_host:port/chroot --delete --topic my_topic_name
   </pre>
-  Topic deletion option is disabled by default. To enable it set the server config
-    <pre class="brush: text;">delete.topic.enable=true</pre>
+  Since 1.0.0, Topic deletion option is enabled by default. If you wish to disable this option, set the following config
+    <pre class="brush: text;">delete.topic.enable=false</pre>
```

Comment:
In my original review, I suggested that we don't need to include information on how to disable it since it's not a common requirement.

Reply:
Ok.  I  just emoved the doc line about "Topic deletion" from ops.html.  Upgrade notes already mention about this change in default behavior.  Pls check.

Reply:
@ijuma pinging for review